package.json

```diff
@@ -25,6 +25,7 @@
     "lint": "yarn lint:scripts && yarn lint:css",
     "lint:scripts": "eslint app/scripts/",
     "lint:css": "stylelint 'app/styles/**/**' 'app/scripts/**/*.(js|ts|tsx|jsx)'",
+    "postinstall": "husky",
```

Comment:
This will run after the install of veda-ui as a registry, which we don't want it. Do we need this @stephenkilbourn

Reply:
@hanbyul-here - ah, I didn't realize that was going to be a consideration.   Looking further at the [husky docs](https://typicode.github.io/husky/how-to.html#manual-setup) for the setup of yarn, we need that postinstall setup, but we also need to disable it before and after the npm publish setup: 
```
{
  "scripts": {
    // Yarn doesn't support prepare script
    "postinstall": "husky",
    // Include this if publishing to npmjs.com
    "prepack": "pinst --disable",
    "postpack": "pinst --enable"
  }
}
```
i'm adding those in a follow-up PR here: #1384